pyBeam/Makefile

```diff
@@ -6,15 +6,52 @@ CODI_PATH = /home/rsanfer/pyBeam/pyBeam/CoDiPack/include
 INCLPATH =  -I$(EIGEN_PATH) -I$(CODI_PATH)
 #==========================================================
 
+_SRC = ./src/ROMFM_FULLORDER.cpp \
+       ./src/ad.cpp \
+       ./src/input.cpp \
+       ./src/geometry.cpp \
+       ./src/FiniteElement.cpp \
+       ./src/StructSyst.cpp \
+       ./src/Rotations.cpp \
+       ./src/beam.cpp
 
-all:
-	swig -c++ -python -Wall -py3 pyBeam.i
-	# I need to add all the requested .cpp files used in the DLM.cpp for the relative .o files... -DDEBG
-	g++ -O2 -c -w -std=gnu++11 -DLINUX=1 -fPIC pyBeam_wrap.cxx ./src/ROMFM_FULLORDER.cpp ./src/input.cpp ./src/FiniteElement.cpp ./src/StructSyst.cpp ./src/Rotations.cpp ./src/beam.cpp -I/usr/include/python3.7m $(INCLPATH)
-	# ... That need to be used here so that the relative pyDLM_Cpp_wrap.o compiles the correct _pyDLM_Cpp.so
-	g++ -O2 -shared -fPIC ROMFM_FULLORDER.o input.o FiniteElement.o StructSyst.o Rotations.o beam.o pyBeam_wrap.o -o _pyBeam.so  $(INCLPATH) 
+_OBJS = ./obj/ROMFM_FULLORDER.o \
```

Comment:
This makes the makefile more readable

Reply:
Great,

I would add a further option (with time) addressing the python specific version we use (if 2.7 or 3.7, which is the one you are using I guess). 

This is necessary in lines:
swig -c++ -python -Wall **-py3** ./swig/pyBeam.i
g++ -O2 -c -w -std=gnu++11 -DLINUX=1 -fPIC ./swig/pyBeam_wrap.cxx $(_SRC) **-I/usr/include/python3.7m** $(INCLPATH)$(INCLPATH)

Reply:
Ok, I'll keep that one in mind. I'll add it to the tab "Projects" (see above)